### PR TITLE
feat(unlock-app) - hotfix sendemail

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
Correctly pass `newtorkId` for send email

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/operations/wedlocksOperations.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ type Params = {
   keychainUrl?: string
   lockName: string
   network: string
+  networkId: number
```
Comment (julien51) [inline locksmith/src/operations/wedlocksOperations.ts:34]: We already have network. Why are you passing networkId too?

**Review by julien51 — CHANGES_REQUESTED** [review]
@kalidiagne Please explain hat you are trying to do and why you are not using `network` ?

**Comment by kalidiagne** [comment]
> @kalidiagne Please explain hat you are trying to do and why you are not using `network` ?

we are passing network name and not the id, we need the id to correctly use it in the queries 

ref: 
```
 networkId: network!,
 network: networks[network!]?.name ?? '',
```

File: locksmith/src/operations/wedlocksOperations.ts (hunk 1 above)
Comment (kalidiagne) [reply locksmith/src/operations/wedlocksOperations.ts:34]: we are passing network name and not the id, we need the id to correctly use it in the queries

ref:

```
 networkId: network!,
 network: networks[network!]?.name ?? '',
 ```

Comment (julien51) [reply locksmith/src/operations/wedlocksOperations.ts:34]: But the templates don't need it! I think we should not pollute the `params` object with stuff that's not needed by the templates...

**Review by julien51 — APPROVED** [review]

**Pull request merged**